src/index.css

```diff
@@ -46,3 +46,35 @@ body {
     max-width: 1200px;
   }
 }
+
+.block_title {
+  font-size: 17px;
+}
```

Comment:
Instead of `block_title`, `card_title`, `user_title`, override bootstrap styles for `h1 ... h6` elements.